code/modules/client/client procs.dm

```diff
@@ -392,6 +395,14 @@
 		if(ckey != query_cid.item[1])
 			related_accounts_cid.Add("[query_cid.item[1]]")
 
+	var/admin_rank = "Player"
+	if(holder)
+		admin_rank = holder.rank
+	// Admins don't get slammed by this, I guess
+	else
+		if(check_randomizer())
```

Comment:
I figure it doesn't matter if a CID-randomized client doesn't get logged because they're not going to be sticking around long if they ARE randomizing, one way or the other